.bumpversion.cfg

```diff
@@ -1,6 +1,19 @@
 [bumpversion]
-current_version = 3.1.0
+current_version = 3.1.3-01
```

Comment:
Use semver, e.g. `3.1.6-dev1`

Reply:
Ok, technically `-01` is also a pre-release per the [semver definition](https://semver.org/), but folks are used to seeing "dev" or "alpha" here too.